simulation_parameters/microbe_stage/biomes.json

```diff
@@ -1569,6 +1579,7 @@
         "name": "Sea Floor",
         "background": "seafloor",
         "icon": "res://assets/textures/gui/bevel/bathyFloor.png",
+        "depth": [6000, 12000],
```

Comment:
I think the depth for the sea floor should be set to be closer to the vents due to the way the current patch map is.